python/lsst/afw/image/photoCalibContinued.py

```diff
@@ -0,0 +1,62 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+"""This file only exists to provide deprecation warnings for the deprecated
+Calib-style interface.
+"""
+
+__all__ = []  # import this module only for its side effects
+
+from lsst.utils import continueClass
+
+from deprecated.sphinx import deprecated
+
+from .photoCalib import PhotoCalib
+
+
+@continueClass  # noqa F811
```

Comment:
I think this should formally be `noqa: F811` (the colon is important to flake8 since it tells it to only mask F811 rather than ignoring all warnings).

Reply:
It looks like we get this wrong all over in afw: I find 33 uses of `noqa ` (space) vs. 8 uses of `noqa: ` (colon space).